apps/tailwind-components/components/input/InputGroupContainer.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <div ref="inputGroup">
+    <slot></slot>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { useFocusWithin } from "@vueuse/core";
```

Comment:
I need some explanation why this doesn't use imply the @focus @blur?
What magic is going on, and is that semantically correct?
AFAIK only checkbox and radio can receive focus, not its container. Is this change of HTML semantic not risky?

Reply:
This checks to see if there are active elements within the container (checkboxes, radio, buttons, other inputs, etc.) and passes the focus and blur events to the parent element(s).

Reply:
why would you need vueuse for that? Is it not simply enough to catch focus/blur events and emit them?

Reply:
Ah, we want to fire focus only at first focus to anything in the group, and only blur if we are outside again.